src/main/services/restoreService.ts

```diff
@@ -314,11 +314,6 @@ export const restoreFromBackup = async (opts: {
 				serviceState.inProgressStateMachine = null;
 				// eslint-disable-next-line no-underscore-dangle
 				const error: ErrorState = JSON.parse(restoreService._state.context.error ?? null);
-				const siteModel = new LocalSiteModel(site);
```

Comment:
Without the `restart` call here, "restore site to this backup" (from the Tools → Cloud Backups page) hangs at the "Restore backup…" step for me when the restore finishes:

```sh
{"thread":"main","class":"BackupAddonRestoreService","message":"getting backup credentials [site id: 33qnTjyHT]","level":"info","timestamp":"2023-09-07T07:34:28.302Z"}
{"thread":"main","class":"BackupAddonRestoreService","message":"creating tmp dir [site id: 33qnTjyHT]","level":"info","timestamp":"2023-09-07T07:34:28.961Z"}
{"thread":"main","class":"BackupAddonRestoreService","message":"restoring backup [site id: 33qnTjyHT]","level":"info","timestamp":"2023-09-07T07:34:28.962Z"}
{"thread":"main","class":"BackupAddonRestoreService","message":"removing the following directories/files to prepare for the site backup: \"/Users/nick.cernis/Local Sites/testbackup/app\", \"/Users/nick.cernis/Local Sites/testbackup/.localbackupaddonignore.txt\"","level":"info","timestamp":"2023-09-07T07:34:44.948Z"}
{"thread":"main","class":"BackupAddonRestoreService","message":"moving site from tmp dir [site id: 33qnTjyHT]","level":"info","timestamp":"2023-09-07T07:34:44.948Z"}
{"thread":"main","class":"BackupAddonRestoreService","message":"Copied '/var/folders/mc/yf37tb4j5bjbwhv6z0m1bcfm0000gp/T/tmp-17506-zRwlYLheUoI1/app' to '/Users/nick.cernis/Local Sites/testbackup/app'","level":"info","timestamp":"2023-09-07T07:34:46.532Z"}
{"thread":"main","class":"BackupAddonRestoreService","message":"Copied '/var/folders/mc/yf37tb4j5bjbwhv6z0m1bcfm0000gp/T/tmp-17506-zRwlYLheUoI1/.localbackupaddonignore.txt' to '/Users/nick.cernis/Local Sites/testbackup/.localbackupaddonignore.txt'","level":"info","timestamp":"2023-09-07T07:34:46.533Z"}
{"thread":"main","class":"BackupAddonRestoreService","message":"restoring database [site id: 33qnTjyHT]","level":"info","timestamp":"2023-09-07T07:34:46.536Z"}
{"thread":"main","service":{},"serviceBinVersion":{},"message":"Database responded to ping.","level":"debug","timestamp":"2023-09-07T07:34:46.548Z"}
{"thread":"main","service":{},"serviceBinVersion":{},"message":"Database responded to ping.","level":"debug","timestamp":"2023-09-07T07:34:46.580Z"}
{"thread":"main","service":{},"serviceBinVersion":{},"message":"Database responded to ping.","level":"debug","timestamp":"2023-09-07T07:34:46.609Z"}
{"thread":"main","service":{},"serviceBinVersion":{},"message":"Database responded to ping.","level":"debug","timestamp":"2023-09-07T07:34:46.901Z"}
{"thread":"main","class":"BackupAddonRestoreService","message":"finished [site id: 33qnTjyHT]","level":"info","timestamp":"2023-09-07T07:34:47.433Z"}
```

Leaving the restore in this state forever (or until I quit Local):

<img width="1312" alt="Screenshot 2023-09-07 at 09 39 30" src="https://github.com/getflywheel/local-addon-backups/assets/647669/9c3b2259-f50c-4439-ba50-0741959d67a3">

If I reinstate the `restart` call (and surrounding removed code here), the backup succeeds.

Reply:
Just noting that this seems intermittent — without the `restart` call, sometimes restore will hang, sometimes it succeeds.

Probably safer to leave the restart calls in place? That way we focus on fixing the Windows bug and don't accidentally introduce regressions.

Reply:
I think that might actually be fixed by adding back the:

`sendIPCEvent('updateSiteStatus', site.id, initialSiteStatus);`? If it's hanging, not sure it'd even reach the `restart()` call though. I didn't run into this but I'll revisit